tests/test_django/urls.py

```
@@ -3,70 +3,255 @@
+            fields={
+                "id": Infer,
+                "name": Infer,
+                "address": Infer,
+            }
+        ),
+    )
 
 
-class BookCrudl(Crudl):
-    """CRUDL for the Book model."""
+class BookCopyCrudl(CrudlController[BookCopy]):
```

Ruff (0.8.2)

Missing docstring in public class

(D101)